### DISCUSSION TITLE
Where's the previous opening in a standalone window option?

### DISCUSSION THREAD

**Discussion by woohling** [reduxjs/redux-devtools#1057] [General]
Now redux devtools can only be opened like this. It was very helpful when this window can be a standalone one so you can watch all the changes alongside with the web page. 
![image](https://user-images.githubusercontent.com/10751371/150931075-7bf94201-5393-4ee4-93d3-54604ac09430.png)

**Comment by Methuselah96** [comment]
The buttons on the bottom right (specifically the grid icons) allow you to open it in separate window.

**Reply by woohling** [reply]
Thanks for the reply! However, a small feedback, the icons are really not intuitive and don't even have a title suggesting what is is when hovering on it 😅
